public/app/plugins/datasource/grafana-azure-monitor-datasource/types/query.ts

```diff
@@ -51,6 +54,11 @@ export interface AzureMetricQuery {
 
   /** @deprecated Remove this once angular is removed */
   allowedTimeGrainsMs?: number[];
+
+  /** @deprecated migrated to dimensionFilters */
+  dimension?: string;
+  /** @deprecated migrated to dimensionFilters */
+  dimensionFilter?: string;
```

Comment:
Why migrating from dimension to dimensionFilter? Is it to align it better with the api?

Reply:
I'm not sure!

This are all old and existing migrations that the Angular query_ctrl.js has been doing for ages, and needs to be moved so they can happen in the react QueryEditor now that Angular is not in the picture any more.